plugin/othttp/handler.go

```diff
@@ -93,6 +98,17 @@ func WithSpanOptions(opts ...trace.StartOption) Option {
 	}
 }
 
+// WithFilter adds a filter to the list of filters used by the handler.
+// If any filter indicates to exclude a request then the request will not be
+// traced. All filters must allow a request to be traced for a Span to be created.
+// Filters will be invoked for each processed request, it is advised to make them
+// simple and fast.
+func WithFilter(f Filter) Option {
```

Comment:
Maybe add one line saying that in the absence of any filter, all requests are traced.

Reply:
Good call.  Seems obvious to me, but I wrote it so of course it would seem obvious. :)  Added a line to the docblock.